app/localization.js

```diff
@@ -152,5 +152,5 @@ export class Localizer {
     }
 }
 
-export let l10n = new Localizer();
+export const l10n = new Localizer();
```

Comment:
squash all these into the previous let/const/var commit, if possible

Reply:
This is the let/const/var commit, or am I missing something?